creator-node/nginx_conf/nginx.conf

```diff
@@ -0,0 +1,44 @@
+worker_processes 1;
+
+events {
+    worker_connections 1024;
+}
+
+http {
+    lua_package_path "/usr/local/openresty/conf/?.lua;;";
+
+    proxy_cache_path /usr/local/openresty/cache levels=1:2 keys_zone=cidcache:1000m
+					max_size=10g inactive=10m use_temp_path=off;
+
+    server {
+        listen 4000;
+
+        # Match the path /ipfs/<cid: string>. If present in cache, serve. 
+        # Else, hit upstream server + update cache + serve.
+        # ^~ : if request matches this route, do not attempt to bypass via the `/` pattern below
+        location ^~ /ipfs {
+            proxy_cache cidcache;
+            proxy_pass http://127.0.0.1:3000;
```

Comment:
Since we can't set an env var for this port either, this is also forced to be port 3000

Reply:
yeah, I think we'll just have to keep these ports hard-coded. for local testing, we should only use the cache for cn1